src/node_lib.h

```diff
@@ -109,7 +117,7 @@ namespace node{ namespace lib {
     /*
     Adds a new JavaScript module to the *running* Node.js engine.
     */
-    v8::MaybeLocal<v8::Object> IncludeModule(const std::string & modul_name);
+    v8::Local<v8::Object> IncludeModule(const std::string & modul_name);

```

Comment:
why is this a Local now? does JS' include() always return something, even if the module is not found?

Reply:
I had made it local to simplify my initial implementation, which lacks error handling anyways :)
However, I haven't gotten around to finish the `IncludeModule` implementation, so feel free to change it back if you work on it.